CHANGELOG.md

```diff
@@ -1,4 +1,93 @@
 
+<!-- 8.1.0-beta1 START -->
+
+# 8.1.0-beta1 (2021-07-22)
+
+### Features and enhancements
+
+* **Alerting:** Add alertmanager notifications tab. [#35759](https://github.com/grafana/grafana/pull/35759), [@nathanrodman](https://github.com/nathanrodman)
+* **Alerting:** add button to deactivate current alertmanager configuration. [#36951](https://github.com/grafana/grafana/pull/36951), [@domasx2](https://github.com/domasx2)
+* **Alerting:** add toggle in Loki/Prom datasource config to opt out of alerting ui. [#36552](https://github.com/grafana/grafana/pull/36552), [@domasx2](https://github.com/domasx2)
+* **Alerting:** allow  any "evaluate for" value >=0 in the alert rule form. [#35807](https://github.com/grafana/grafana/pull/35807), [@domasx2](https://github.com/domasx2)
+* **Alerting:** if cortex alertmanager returns empty user config, load default config from status endpoint. [#35769](https://github.com/grafana/grafana/pull/35769), [@domasx2](https://github.com/domasx2)
+* **Alerting:** view to display alert rule and its underlying data. [#35546](https://github.com/grafana/grafana/pull/35546), [@mckn](https://github.com/mckn)
+* **Annotation Panel:** release annotation panel. [#36959](https://github.com/grafana/grafana/pull/36959), [@ryantxu](https://github.com/ryantxu)
+* **Annotations:** Add typeahead support for tags in builtin annotations. [#36377](https://github.com/grafana/grafana/pull/36377), [@ashharrison90](https://github.com/ashharrison90)
+* **AzureMonitor:** Add curated dashboards for Azure services. [#35356](https://github.com/grafana/grafana/pull/35356), [@avidhanju](https://github.com/avidhanju)
+* **AzureMonitor:** Add support for deep links to Azure Portal for Metrics. [#32273](https://github.com/grafana/grafana/pull/32273), [@shuotli](https://github.com/shuotli)
+* **AzureMonitor:** Curated dashboards cont. [#35959](https://github.com/grafana/grafana/pull/35959), [@shuotli](https://github.com/shuotli)
```

Comment:
@joshhunt, this title needs to contain stand-alone information that the reader can understand without opening the PR.

Reply:
This PR was not actually merged - this line should be removed.

I guess the PR bot doesnt check if it was actually merged or not?

I removed the label. Will the changelog be regenerated or should I edit this manually?

Reply:
They will be regenerated.